pulsar-common/src/main/java/org/apache/pulsar/common/events/PulsarEvent.java

```diff
@@ -35,4 +37,9 @@
     private EventType eventType;
     private ActionType actionType;
     private TopicPoliciesEvent topicPoliciesEvent;
+    /**
+     * Which remote clusters to replicate to
+     */
+    private List<String> replicateTo;
```

Comment:
Should be `Set<String> replicateTo`?

Reply:
And I think we can avoid adding `replicateTo` in PulsarEvent? instead, we can introduce `isGlobal` in TopicPoliciesEvent? TopicPolicies already have `isGlobal` information, so that we can use it directly in the `TopicPoliciesSystemTopicClient.java`, it looks like

```
if (event.policies.isGlobal) {
    .replicationClusters(null);
} else {
     .replicationClusters(emptySet);
}
```

Reply:
We at least need to keep `replicateTo`, when deleting Topic Policies, the message body is null